src/views/Nft/market/Profile/components/ActivityHistory/index.tsx

```diff
@@ -28,9 +29,9 @@ const ActivityHistory = () => {
   const { t } = useTranslation()
   const [currentPage, setCurrentPage] = useState(1)
   const [maxPage, setMaxPages] = useState(1)
-  const [activitiesSlice, setActivitiesSlice] = useState<Activity[]>([])
+  const [activitySlice, setActivitySlice] = useState<Activity[]>([])
   const [nftMetadata, setNftMetadata] = useState<NftToken[]>([])
-  const [sortedUserActivities, setSortedUserActivities] = useState<Activity[]>([])
+  const [sortedUserActivity, setSortedUserActivity] = useState<Activity[]>([])
```

Comment:
Not sure why rename Activities -> Activity. It's an array so plural is more appropriate, no? 😄

Reply:
Done